src/integrationTest/java/uk/gov/hmcts/reform/judicialapi/RefreshUserProfileIntegrationTest.java

```diff
@@ -99,12 +100,11 @@ void shouldReturn_200_ValidParameters_ccdPageSize(String role) {
                 .objectIds(Collections.emptyList())
                 .build();
         // pageSize 2
-        var response = judicialReferenceDataClient.refreshUserProfile(refreshRoleRequest,2,
+        var errorResponseMap = judicialReferenceDataClient.refreshUserProfile(refreshRoleRequest,2,
                 0,"", "objectId", role, false);
-        assertThat(response).containsEntry("http_status", "200 OK");
-
-        var userProfileList = (List<?>) response.get("body");
-        assertThat(userProfileList).hasSize(2);
+        assertThat(errorResponseMap).containsEntry("http_status", "400");
```

Comment:
Looks like you have removed a success scenario and replaced with negative scenario. Now there are no success scenarios for refresh api. Can you add this as a separate test ?